git-coco

```diff
@@ -164,6 +192,50 @@ pr() {
     echo "https://console.aws.amazon.com/codesuite/codecommit/repositories/${repo}/pull-requests/${id}/changes"
 }
 
+prlist() {
+    local repo list_of_prs pull_info
+    local dst_branch src_branch is_merge pull_id author title
+
+    repo="$1"
+
+    # Handle default case
+    if [ -z "$repo" ]; then
+        local url=$(echo $(git remote -v) | awk -F ' ' '{print $2}')
```

Comment:
This could be better written as:

```suggestion
        local url=$(git remote -v | awk -F ' ' '{ print $2; exit; }')
```

Which removes some of the coincidental feel of unquoted echo not expanding newlines.

Another option would be to loop through all the remotes until `url_parser` returned something.

Reply:
Went for the loop implementation. Code now loops through each URL until a valid repository name is found. If not, it exits with an error.